.github/workflows/Tests.yml

```diff
@@ -172,99 +166,15 @@ jobs:
       if: ${{ always() }}
 
 
-  COVERAGE-MATS:
+  COVERAGE:
```

Comment:
LGTM! We are now dropping support for pre-3.9 (as part of #134) Makes sense to consolidate to a single COVERAGE job.